src/plugins/dashboard/public/dashboard_container/embeddable/api/add_panel_from_library.ts

```diff
@@ -21,12 +22,17 @@ export function addFromLibrary(this: DashboardContainer) {
     settings: { uiSettings, theme },
     embeddable: { getEmbeddableFactories, getEmbeddableFactory },
     http,
+    savedObjectsManagement,
   } = pluginServices.getServices();
 
   if (isErrorEmbeddable(this)) return;
   this.openOverlay(
     openAddPanelFlyout({
-      SavedObjectFinder: getSavedObjectFinder(uiSettings, http as HttpStart),
+      SavedObjectFinder: getSavedObjectFinder(
+        uiSettings,
+        http as HttpStart,
+        savedObjectsManagement as unknown as SavedObjectsManagementPluginStart
```

Comment:
Thanks for changing savedObjectsManagementService type to match interface of getSavedObjectFinder. Cast to `unknown` is no longer needed here